source/ed/types.h

```diff
@@ -187,7 +187,7 @@ struct CommercialMode : public Header, Nameable{
 
 struct PhysicalMode : public Header, Nameable{
     const static nt::Type_e type = nt::Type_e::PhysicalMode;
-    double co2_emission = 0.;
+    double co2_emission = -1.;
```

Comment:
I don't understand, in ed you store it in physical_mode and in commercial_mode in navitia.